tests/integration/hdf5/test_retinotopy.py

```diff
@@ -0,0 +1,34 @@
+# import numpy as np
```

Comment:
Should this test be removed since it is commented out?

Reply:
Since this is minor, we can remove it in another PR if you wish. I will try to remedy the retinotopy schema and API for the next minor release.